### PR TITLE
work in progress project1 submission/ top half complete using inline-block value

### PR DESCRIPTION
**Pull request by erikAlon**

### CODE REVIEW

**Comment by erikAlon** [comment]
Name: Erik Alonso

File: project1/css/myIndex2.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+/******************************************TOP***************************************/
+
+body {
+    margin: 25px 500px;
+}
+
+header {
+    /* border: solid yellow; */
```
Comment (nomyard) [inline project1/css/myIndex2.css]: It's best practice to clean up comments when pushing code. 

This goes for random spacing, M.I.A. semi-colons, etc.

File: project1/css/myIndex2.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+nav {
+    /* border: solid yellow; */
```
Comment (nomyard) [inline project1/css/myIndex2.css]: Comment clean up

**Comment by nomyard** [comment]
Hey Erik, just got done reviewing your code and everything looks really well done. Just one thing I would work on is making sure the code you push is as clean as possible when sent in for review. This goes for class work, open source contribution, and workplace contribution. It's always best practice to give the cleanest code possible. Think of it like the final draft of an essay you wrote. Other than that your code looked great! Keep up the good work! 

-Ray

**Comment by nomyard** [comment]
Please close this Pull Request when read- thanks!

**Pull request closed without merging**